projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -92,6 +93,15 @@ export class UpdateChanges {
         }
     }
 
+    /** Add condition funtion. */
+    public addCondition(conditionName: string, callback: Function) {
```

Comment:
This can be described as `callback: (ownerMatch: string) => boolean` for consuming code intellisense

Reply:
Done.